src/structs.rs

```diff
@@ -121,3 +123,13 @@ pub enum PubKey {
     Secp256k1Bytes(Box<[u8]>),
     Secp256k1(secp256k1::PublicKey),
 }
+
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+
+pub struct CborData {
+    /// Number of bytes added to pad to align zfec chunks and bao slices.
+    pub str_name: String,
+    pub str_date: String,
+    pub str_mime_type: String,
```

Comment:
Let's remove the str prefix:

```suggestion
    pub name: String,
    pub date: String,
    pub mime_type: String,
```